docs/source/tutorials/multifidelity/README.rst

```diff
@@ -8,7 +8,7 @@ between different methods, and how to choose the best approach for your own
 problems.
 
 .. note::
-   This tutorial makes use of the *benchmarking* framework, you need to have
+   In order to run the code in this tutorial, you need to have
    `installed Syne Tune from source <../../getting_started.html#installation>`__
    to run it.
```

Comment:
What components does this tutorial still need that requires it to be installed from source? 

(Fine to leave it like this, but want to verify that indeed it still needs to be installed from source.)

Reply:
Good catch, this is wrong.

Reply:
I'll go over all tutorials and check.

Reply:
I'll also make sure that all benchmarking examples which do not depend on training code in `benchmarking`, use local imports, which means that only their own code has to be installed, not all of `benchmarking`.